__config__/cfn/cloudformation.yml

```diff
@@ -169,14 +169,20 @@ Resources:
           apt-get install -y nodejs
           apt-get install -y yarn
 
+          # nvm
+          curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | bash
+          export NVM_DIR="$HOME/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+
           git clone https://github.com/guardian/guui.git
           cd guui
-          git fetch
-          git checkout sa-productionise
 
           export TERM=xterm-256color
 
-          make build start
+          nvm install 8.9
+          nvm use 8.9
+          make start
```

Comment:
We'll still need to build here
```
make build start
```

Reply:
+1 fixed

Reply:
i guess we need to get travis doign something with the assets it makes